tests/WebTest.php

```diff
@@ -192,6 +199,22 @@ public function testCookieHttpOnly()
         $this->assertEquals(true, $this->getCookie()->getHttpOnly());
     }
 
+    public function testSessionIDReuse()
+    {
+        $this->request("set.php?key=ok&value=yep");
+
+        $cookie = $this->getCookie();
+        $sessionID = $cookie->getValue();
+
+        $session = new SessionInstance(
+            $cookie->getName(),
+            null,
+            $sessionID
+        );
+
+        $this->assertEquals("yep", $session->getSet('ok'));
```

Comment:
Session values are created in PHP SAPI web server using request to set.php

Here, the session is accessed from CLI, a very different place from web server. This is to prove that session is accessible irrespective of place where it's created.